deviceModeInit.js

```diff
@@ -85,11 +85,17 @@ var rudderTracking = (function () {
         : "";
     fetchCart()
       .then((cart) => {
-        if (!checkCartAttributes(cart) || checkUpdateTime(cart)) {
+        const needToUpdateCart = checkCartAttributes(cart);
```

Comment:
The function name `checkCartAttributes` does not match with behaviour change either name  or behaviour

Reply:
Done